csv_editor.py

```diff
@@ -129,7 +129,8 @@ def daily_check_in_auto(self, users):
         if(is_working_day):
             yy, mm, dd = current_time.year,current_time.month,current_time.day
             for user in users:
-
+                if user in inactive_user:
```

Comment:
It's better removing inactive_user in users rather than checking inactive_user in for loop.

Reply:
Done!